src/raven_error_logger.erl

```diff
@@ -197,15 +197,16 @@ parse_message(Level, Pid, "Error: ~p" ++ _ = Format, [{failed, Reason, Extras} |
 			[ {Key, Value} || {Key, Value} <- Extras, is_atom(Key) ]
 		]}
 	]};
-parse_message(Level, Pid, "Catched this: ~p" ++ _ = Format,
-			  [{{Class, Reason}, [{_, _, _, _} | _] = Stacktrace} | Rest])
+parse_message(Level, Pid, "Exception: ~p\n"
+						  "Extras: ~p" = Format,
```

Comment:
This looks weird?

Reply:
Yeah, this file uses tabs... :/ Looks okay in the editor, trust me :)